bin/tic_tac_toe

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/ruby
+
+require './lib/player'
+require './lib/cli_interface'
+require './lib/human'
+require './lib/computer'
+require './lib/board'
+require './lib/game_play'
+require './lib/marks'
+require './lib/players_factory'
+
+interface = CliInterface.new(Kernel, Kernel, "./lang/")
+board = Board.new
+game_play = GamePlay.new(board, interface)
+
+game_play.language
```

Comment:
Those lines look quite procedural.
It's good that they are unit tested separately but do you think there's a way to minimize the effort needed in here to start a new game?